deltachat-ios/Controller/ContextMenuController.swift

```diff
@@ -23,14 +23,14 @@ class ContextMenuController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
 
-        let viewType = msg.viewtype
+        let viewType = msg.type
         var thumbnailView: UIView?
         switch viewType {
-        case .image:
+        case DC_MSG_IMAGE, DC_MSG_STICKER:
```

Comment:
Using a `switch` on `Int` feels a bit un-swifty when there're Enums :D

Reply:
the defines vome direcly from the bindings, no effort. maintaining an additional swift enum (or generating them somehow) seems not worth the effort. worse, it led to the bugs we had as it was not maintained correctly.

we're now just doing the same for the type as for all other defines